libunwind/src/UnwindRegistersSave.S

```diff
@@ -897,8 +884,21 @@ DEFINE_LIBUNWIND_FUNCTION(__unw_getcontext)
   stp    d28,d29, [c0, #0x0e0]
   str    d30,     [c0, #0x0f0]
   str    d31,     [c0, #0x0f8]
+#ifdef _LIBUNWIND_HAS_CHERI_LIB_C18N
+  // Store the trusted stack pointer
+  stp    c0, c30, [csp, #-0x20]!
+  mov    x0, xzr
```

Comment:
Is there a reason not to pass c30 here and let it automatically do the right thing, beyond the overhead of doing the trampoline check?

Reply:
No, but I can certainly change it to pass c30.